pkg/build/builder/cmd/dockercfg/cfg_test.go

```diff
@@ -3,11 +3,56 @@ package dockercfg
 import (
 	"io/ioutil"
 	"os"
+	"path/filepath"
 	"testing"
 
 	"k8s.io/kubernetes/pkg/credentialprovider"
 )
 
+func TestGetDockerAuth(t *testing.T) {
+	var (
+		workingDirPath     = ""
+		homeDirPath        = os.Getenv("HOME")
+		rootDirPath        = "/"
+		configJsonFileName = "config.json"
+	)
+	var fileInfo *os.File
+
+	content := "{ \"auths\": { \"test-server-1.tld\":{\"auth\":\"Zm9vOmJhcgo=\",\"email\":\"test@email.test.com\"}}}"
+
+	dockerConfigJsonLocations := []string{workingDirPath, homeDirPath, rootDirPath}
```

Comment:
@liggitt reasonable! I'll delete the home dir. May be using "/tmp" dir is better. what do you think?